Assets/Scripts/CountdownController.cs

```diff
@@ -5,9 +5,21 @@
 
 public class CountdownController : MonoBehaviour
 {
-    private static float countdownTime = 3f;
+    public static CountdownController Instance { get; private set; }
+    public float countdownTime = 3f;
     public Text countdownDisplay;
 
+    private void Awake()
+    {
+        if (Instance != null && Instance != this)
```

Comment:
that's nice practice but you don't need to do that. In fact if you want to destroy additional instances just make sure to copy the values of the instance so that changes made in the inspector will actually affect the scene otherwise older values from another scene will persist.

Note that this isn't a big issue especially in this specific case